downstream/modules/platform/proc-completing-post-installation-tasks.adoc

```diff
@@ -1,54 +1,64 @@
 [id="completing-post-installation-tasks_{context}"]
 
-= Completing Post Installation Tasks
+= Completing post installation tasks
 
-== Adding Controller Subscription
+[role="_abstract"]
+After you have completed the installation, review the post-installation tasks in this section to ensure that {HubName} and {ControllerName} deploy properly.
+
+== Adding a controller subscription
 
 .Procedure
 
-. Navigate to the FQDN of the {ControllerNameStart}. Login with admin and the password you specified as `admin_password` in your inventory file.
+. Navigate to the FQDN of the {ControllerNameStart}. Log in with the username admin and the password you specified as `admin_password` in your inventory file.
 
 . Click btn:[Browse] and select the __manifest.zip__ you created earlier.
 
 . Click btn:[Next].
 
-. Uncheck btn:[User analytics] and btn:[Automation analytics]. These rely on an Internet connection and should be turned off.
+. Uncheck btn:[User analytics] and btn:[Automation analytics]. These rely on an internet connection and should be turned off.
 
 . Click btn:[Next].
 
 . Read the End User License Agreement and click btn:[Submit] if you agree.
 
 == Updating the CA trust store
 
-=== Self-Signed Certificates
```

Comment:
Suggestion: Change the content in this section to procedure/task format.

Reply:
I need to double check with @rogrange, but I *think* that the content on adding a controller subscription is duplicated [here](https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/getting_started_with_automation_controller/controller-managing-subscriptions#doc-wrapper), and the steps in Updating the CA trust store are taken from t[his knowledgebase article](https://access.redhat.com/solutions/6707451), so we may be able to just link to that content and shorten or even eliminate this module.

Reply:
@hherbly , yse, you are correct, that section was taken from the KB article. We saw no reason to reinvent the wheel, when it was already available. to us. :-)  If you can trim down the section, that might suffice. Since this is for a disconnected installation, my thought was to give them as much information as possible within the document since they would be working offline for the install.